cpp/memilio/math/integrator.h

```diff
@@ -112,8 +162,9 @@ class OdeIntegrator
         bool step_okay = true;
 
         FP dt_copy; // used to check whether step sizing is adaptive
-        FP dt_restore = 0; // used to restore dt if dt was decreased to reach tmax
-        FP t          = t0;
+        FP dt_restore  = 0.0; // used to restore dt if dt was decreased to reach tmax
```

Comment:
Maybe adapt comment as dt_min_copy is also "used to restore dt if dt was decreased to reach tmax"?

Reply:
Yes, though it restores dt_min. Should it be renamed to dt_min_restore as well? I think _copy is okay, and shorter.

Reply:
Maybe rename dt_restore to dt_copy then? I think it makes sense to name variables that do the same thing similarly, just for dt and dt_min.

Reply:
Ok. If you are happy with the current naming, leave it as it is :)